tests/DCPS/Observer/Domain.h

```diff
@@ -11,8 +11,13 @@ class Domain {
 
   template<typename Qos>
   static void change_qos(Qos& qos, const std::string& data) {
+#if __cplusplus >= 199707L
```

Comment:
Similar to the note above: I don't know of anything different based on the C++ versions.  The Standard Library types always use `size_t` and the IDL Sequences (C++ classic mapping) use a 32-bit unsigned integer.